src/RedisLockingStrategy.php

```diff
@@ -92,6 +92,7 @@ public function __construct($subject)
         $this->value = uniqid();
 
         $this->backend = $this->connectBackend($configuration);
+        $this->logger = GeneralUtility::makeInstance(Logger::class, '');
```

Comment:
sorry, the correct way is to makeInstance of LogManager, calling ->getLogger(__CLASS__)